doc/en/api/1.0.0/datastores.yaml

```diff
@@ -290,22 +285,609 @@ paths:
+        | EntityResolver | Sets the entity resolver used to expand XML entities | program | EntityResolver | False | `org.geotools.xml.PreventLocalEntityResolver@75e98519` |
+        | Time-out | This allows the user to specify a timeout in milliseconds. This param has a default value of 3000ms. | user | Integer | False | `3000` |
+        | GmlComplianceLevel | Optional OGC GML compliance level required. | user | Integer | False | `0` |
+        | Lenient | Indicates that datastore should do its best to create features from the provided data even if it does not accurately match the schema.  Errors will be logged but the parsing will continue if this is true.  Default is false | user | Boolean | False | `False` |
+        | Password | This allows the user to specify a username. This param should not be used without the USERNAME param. | user | String | False | ` ` |
+        | Use Default SRS | Use always the declared DefaultSRS for requests and reproject locally if necessary | advanced | Boolean | False | `False` |
+        | Namespace | Override the original WFS type name namespaces | advanced | String | False | ` ` |
+        | Username | This allows the user to specify a username. This param should not be used without the PASSWORD param. | user | String | False | ` ` |
+        | Axis Order Filter | Indicates axis order used by the remote WFS server for filters. It applies only to WFS 1.x.0 servers. Default is the same as AXIS_ORDER | advanced | String | False | ` ` |
+        | GmlCompatibleTypeNames | Use Gml Compatible TypeNames (replace : by _). | user | Boolean | False | `False` |
+        | Maximum features | Positive integer used as a hard limit for the amount of Features to retrieve for each FeatureType. A value of zero or not providing this parameter means no limit. | user | Integer | False | `0` |
+        | Axis Order | Indicates axis order used by the remote WFS server in result coordinates. It applies only to WFS 1.x.0 servers. Default is Compliant | advanced | String | False | `Compliant` |
+        | WFS Strategy | Override wfs stragegy with either cubwerx, ionic, mapserver, geoserver, strict, nonstrict or arcgis strategy. | user | String | False | `auto` |
+        | Try GZIP | Indicates that datastore should use gzip to transfer data if the server supports it. Default is true | user | Boolean | False | `True` |
+        | Encoding | This allows the user to specify the character encoding of the XML-Requests sent to the Server. Defaults to UTF-8 | user | String | False | `UTF-8` |
+        | Outputformat | This allows the user to specify an outputFormat, different from the default one. | advanced | String | False | ` ` |
+
+    in: body
+    required: true
+    schema:
+      $ref: "#/definitions/datastore"
+  dataStorePut:
+    name: dataStoreBody
+    description: |
+      The updated data store definition.
+      For a PUT, only values which should be changed need to be included.
```

Comment:
idea: clarification that updating an individual connection parameter is not possible, the entire connection parameter block would need to be changed?

Reply:
I did not actually know that was the case, but verified in the code. I suppose that makes sense, as otherwise there is no way to remove connection parameters.

Will add clarification.